models/pianocktail_cnn.py

```diff
@@ -18,24 +18,33 @@ def __init__(self, name=config.ModelEnum.PIANOCKTAIL_CNN.value, subspectrogram_p
 
         self.pooling1 = AveragePooling2D(pool_size=2,name=f"{name}_pool1")
 
+        self.dropout1 = Dropout(0.1,name=f"{name}_dropout1")
+
         self.conv2 = Conv2D(filters=16,kernel_size=5,activation=tf.nn.relu,name=f"{name}_conv2")
 
         self.pooling2 = AveragePooling2D(pool_size=2,name=f"{name}_pool2")
 
+        self.dropout2 = Dropout(0.1,name=f"{name}_dropout2")
+
         self.flatten = Flatten(name=f"{name}_flatten")
 
         self.dense1 = Dense(200,activation=tf.nn.relu,name=f"{name}_dense1")
 
+        self.dropout3 = Dropout(0.1,name=f"{name}_dropout3")
+
         self.dense2 = Dense(config.NUMBER_OF_EMOTIONS, activation=tf.nn.sigmoid, name=f"{name}_output")
 
     def call(self, inputs, training=False):
         net = self.input_layer(inputs)
         net = self.reshape(net)
         net = self.conv1(net)
         net = self.pooling1(net)
+        net = self.dropout1(net)
```

Comment:
droupout must only be called on training, if you don't set the training parameter, you will also get dropout during testing

Reply:
Good catch